py/torch_tensorrt/dynamo/conversion/_TRTInterpreter.py

```diff
@@ -329,6 +329,32 @@ def _populate_trt_builder_config(
         if self.compilation_settings.enable_weight_streaming:
             builder_config.set_flag(trt.BuilderFlag.WEIGHT_STREAMING)
 
+        if version.parse(trt.__version__) >= version.parse("10.8"):
+            if self.compilation_settings.tiling_optimization_level == 0:
+                builder_config.tiling_optimization_level = (
+                    trt.TilingOptimizationLevel.NONE
+                )
+            elif self.compilation_settings.tiling_optimization_level == 1:
+                builder_config.tiling_optimization_level = (
+                    trt.TilingOptimizationLevel.FAST
+                )
+            elif self.compilation_settings.tiling_optimization_level == 2:
+                builder_config.tiling_optimization_level = (
+                    trt.TilingOptimizationLevel.MODERATE
+                )
+            elif self.compilation_settings.tiling_optimization_level == 3:
+                builder_config.tiling_optimization_level = (
+                    trt.TilingOptimizationLevel.FULL
+                )
+            else:
+                raise ValueError(
+                    f"Invalid tiling optimization level: {self.compilation_settings.tiling_optimization_level}. A valid value should be in [0, 1, 2, 3]."
+                )
+
+            builder_config.l2_limit_for_tiling = (
+                self.compilation_settings.l2_limit_for_tiling
+            )
```

Comment:
if you want to be really safe (when we remove version guarding), you can check if self.compilation_settings.get("l2_limit_for_tiling", -1) != -1 or something.